corehq/apps/data_dictionary/static/data_dictionary/js/data_dictionary.js

```diff
@@ -3,43 +3,66 @@ hqDefine('data_dictionary/js/data_dictionary.js', function () {
 
     var CaseType = function (name) {
         var self = this;
-        self.name = ko.observable(name);
+        self.name = name;
         self.properties = ko.observableArray();
 
         self.init = function (group_dict, changeSaveButton) {
             _.each(group_dict, function (properties, group) {
                 var groupObj = new PropertyListItem(group, true, group, self.name);
                 self.properties.push(groupObj);
                 _.each(properties, function (prop) {
-                    var propObj = new PropertyListItem(prop.name, false, prop.group, self.name, prop.data_type, prop.description);
+                    var propObj = new PropertyListItem(prop.name, false, prop.group, self.name, prop.data_type, prop.description, prop.deprecated);
                     propObj.description.subscribe(changeSaveButton);
                     propObj.dataType.subscribe(changeSaveButton);
+                    propObj.deprecated.subscribe(changeSaveButton);
                     self.properties.push(propObj);
                 });
             });
         };
     };
 
-    var PropertyListItem = function (name, isGroup, groupName, caseType, dataType, description) {
+    var PropertyListItem = function (name, isGroup, groupName, caseType, dataType, description, deprecated) {
         var self = this;
-        self.name = ko.observable(name);
```

Comment:
its static, so it seemed unnecessary to have it be an observable, and i think it was complicating an edge case somewhere in the page